signatures/rainbow/rainbow-compressed/param/IIIc.yaml

```diff
@@ -0,0 +1,12 @@
+name: Rainbow-IIIc
+security level:
+  classical: 217
+  quantum: 167
+  nist category: 3
```

Comment:
they say it satisfies 3 and 4, so because 3<4 (as specified by nist) we can put 4 here, I'd say